docs/csharp/csharp-6.md

```diff
@@ -1,17 +1,570 @@
+qualifying the `Math` class. The `Math` class is a great use case for
+this feature because it does not contain any
+instance methods. You can also use `using static` to import a
+class' static methods for a class that has both static
+and instance methods. One of the most useful examples 
+`System.String`:
+
+[!code-csharp[UsingStatic](../../samples/snippets/csharp/new-in-6/newcode.cs#UsingStatic)]
+
+> [!NOTE]
+> You must use the fully qualified class name, `System.String`
+> in a static using statement. 
+> You can't use the `string` keyword instead. 
+
+You can now call static methods defined in the `String` class without
+qualifying those methods as members of that class:
+
+[!code-csharp[UsingStaticString](../../samples/snippets/csharp/new-in-6/newcode.cs#UsingStaticString)]
+
+The `static using` feature and extension methods interact in
+interesting ways, and the language design included some rules
+the specifically address those interactions. The goal is to
+minimize any chances of breaking changes in existing codebases,
+including yours.
+
+Extension methods are only in scope when called using the
+extension method invocation syntax, not when called as a static method.
+You'll often see this in LINQ queries. You can import the LINQ pattern
+by importing `System.Linq.Enumerable`. 
+
+[!code-csharp[UsingStaticLinq](../../samples/snippets/csharp/new-in-6/newcode.cs#usingStaticLinq)]
+
+This imports all the methods in the `System.Linq.Enumerable` class.
+However, the extension methods are only in scope when called as extension
+methods. They are not in scope if they are called as though they are static
+methods:
+
+[!code-csharp[UsingStaticLinqMethod](../../samples/snippets/csharp/new-in-6/newcode.cs#UsingStaticLinkMethod)]
+
+This decision is because extension methods are typically called using
+extension method invocation expressions. In the rare case where they are
+called using the static method call syntax it is to resolve ambiguity.
+Requiring the class name as part of the invocation seems wise.
+
+There's one last feature of `static using`. The `static using` directive
+also imports any nested types. That enables you to reference any nested
+types without qualification.
+
+## Null-conditional operators
+
+Null values complicate code. You need to check every access
+of variables to ensure you are not dereferencing `null`. The
+***null conditional operator*** makes those checks much easier
+and fluid.
+
+Simply replace the member access `.` with `?.`:
+
+[!code-csharp[NullConditional](../../samples/snippets/csharp/new-in-6/program.cs#NullConditional)]
+
+In the above example, the variable `first` is assigned `null` if the person object
+is null. Otherwise, it gets assigned the value of the `FirstName` property. Most importantly,
+the `?.` means that the above line of code does not generate a `NullReferenceException` when
+the `person` variable is null. Instead, it short-circuits and produces null.
```

Comment:
make sure all the instances where null should not be localized are styled as inline code.